### ISSUE TITLE
Input field prefix and suffix

### ISSUE THREAD

**Issue by dorthrithil** [vmware/clarity#1665]
**Select one ...**  (check one with "x")
```
[ ] bug
[ ] feature request
[x] enhancement
```

### Expected behavior
It would be really nice to have the possibility of adding pre- and suffixes to input fields. This is useful for adding units to an input for example. 

Here's a concrete example of what I mean: https://material.angular.io/components/form-field/overview#prefix-amp-suffix

### Environment details

* **Clarity version:** 0.10.11

**Comment by adityarb88** [comment]
@dorthrithil: Thanks for the feedback. We will pass this back to our UX team and see what they say. 

@reddolan: Could you please take a look at this?

**Comment by reddolan** [comment]
@dorthrithil this looks very similar to the request in #1478. Is that correct?

**Comment by dorthrithil** [comment]
@reddolan Yes, kind of. Although @srivathsah was only referring to the bootstrap input group buttons. The feature I was referring to would be the `input-group-addon` span in bootstrap.
